app.js

```diff
@@ -3,7 +3,8 @@ const express = require('express');
 const path = require('path');
 const logger = require('morgan');
 const app = express();
-
+const indexRoute = require('./routes/index');
+const pizzaRoute = require('./routes/pizza');
```

Comment:
Variables not necessary when they're one time uses. You can just write them inline like so:
```
app.use('/', require('./routes/index'));
```